src/main/java/com/minecolonies/coremod/items/ItemScepterBeekeeper.java

```diff
@@ -0,0 +1,116 @@
+package com.minecolonies.coremod.items;
+
+import com.ldtteam.structurize.util.LanguageHandler;
+import com.minecolonies.api.colony.IColony;
+import com.minecolonies.api.colony.IColonyManager;
+import com.minecolonies.api.colony.buildings.IBuilding;
+import com.minecolonies.api.util.BlockPosUtil;
+import com.minecolonies.api.util.NBTUtils;
+import com.minecolonies.api.util.constant.Constants;
+import com.minecolonies.coremod.colony.buildings.workerbuildings.BuildingBeekeeper;
+import net.minecraft.block.BeehiveBlock;
+import net.minecraft.entity.player.PlayerEntity;
+import net.minecraft.item.ItemStack;
+import net.minecraft.item.ItemUseContext;
+import net.minecraft.nbt.CompoundNBT;
+import net.minecraft.nbt.NBTUtil;
+import net.minecraft.util.ActionResultType;
+import net.minecraft.util.math.BlockPos;
+import org.jetbrains.annotations.NotNull;
+
+import java.util.Collection;
+import java.util.Set;
+import java.util.stream.Collectors;
+
+import static com.minecolonies.api.util.constant.NbtTagConstants.TAG_ID;
+import static com.minecolonies.api.util.constant.NbtTagConstants.TAG_POS;
+
+/**
+ * Beekeeper Scepter Item class. Used to give tasks to Beekeeper.
+ */
+public class ItemScepterBeekeeper extends AbstractItemMinecolonies
+{
+    private static final String NBT_HIVE_POS = Constants.MOD_ID + ":" + "hives";
+
+    /**
+     * BeekeeperScepter constructor. Sets max stack to 1, like other tools.
+     * @param properties the properties.
+     */
+    public ItemScepterBeekeeper(final Properties properties)
+    {
+        super("scepterbeekeeper", properties.maxStackSize(1));
+    }
+
+    @Override
+    public ActionResultType onItemUse(final ItemUseContext useContext)
+    {
+        // if server world, do nothing
+        if (useContext.getWorld().isRemote)
+        {
+            return ActionResultType.FAIL;
```

Comment:
Should probably be pass and not fail.

Reply:
it's copied from lumberjack so I thought this is right

Reply:
Ah, then you can keep it. There it works fine.